channels/in.m3u

```diff
@@ -375,6 +381,8 @@ https://a.jsrdn.com/broadcast/447912f76b/+0000/c.m3u8
 https://live.mycast.in/encode/ee0c5a36ff5a7083ee044991974ad3ba.sdp/playlist.m3u8
 #EXTINF:-1 tvg-id="PratidinTime.in" tvg-name="Pratidin Time" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="News",Pratidin Time (360p)
 https://rtmp.smartstream.video/pratidintime/pratidintime/playlist.m3u8
+#EXTINF:-1 tvg-id="PravasiTV.in" tvg-name="Pravasi TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="" group-title="News",Pravasi TV
+https://5ee50688d7b5d.streamlock.net:444/live/pravasi/playlist.m3u8
```

Comment:
Doesn't seem to work for me.

Reply:
It was working fine. Not working now. can you remove it?

Reply:
I'll wait and try again tomorrow, might simply be Not 24/7.

Reply:
Not playing in the source website also. Error Code: 232404)

Reply:
Still not working right now. Deleting it.